packages/design-system/src/components/ButtonGroup/style.scss

```diff
@@ -25,6 +24,10 @@
       background-color: $ui-background-color;
     }
 
+    &:focus, &:active {
+      box-shadow: none;
```

Comment:
yeah, it resets box-shadow but we don't have any alternative in ButtonGroup to indicate the button was focused. WCAG requires that the user is notified about a focused element

Reply:
So I kept the `box-shadow` and simply improved its display to avoid "cuts" like these:

<img width="454" alt="Zrzut ekranu 2020-02-7 o 15 44 53" src="https://user-images.githubusercontent.com/46320952/74038718-e7385a80-49c0-11ea-9f61-da4c97fc9e19.png">

is that good ?

Reply:
Now it's this:
<img width="392" alt="Zrzut ekranu 2020-02-7 o 15 47 39" src="https://user-images.githubusercontent.com/46320952/74038896-34b4c780-49c1-11ea-9bf0-d27890c0689d.png">

Reply:
Awesome! I am glad you were able to improve it like that.